app/components/post_list/index.tsx

```diff
@@ -312,11 +313,11 @@ const PostList = ({
         );
     }, [currentTimezone, highlightPinnedOrSaved, isTimezoneEnabled, orderedPosts, shouldRenderReplyButton, theme]);
 
-    const scrollToIndex = useCallback((index: number, animated = true) => {
+    const scrollToIndex = useCallback((index: number, animated = true, applyOffset = true) => {
         listRef.current?.scrollToIndex({
             animated,
             index,
-            viewOffset: 0,
+            viewOffset: applyOffset ? Platform.select({ios: -45, default: 0}) : 0,
```

Comment:
-45 stands for ... ?

Reply:
I'll add a constant instead but this is the amount of space that the list is sitting behind the navigation bar